### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/11/day10.html

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#19] [Comments]
# https://mnfienen.github.io/talulat/2023/08/11/day10.html

Day 10 - Montreal y’all! miles today: 62 total miles: 596 km today: 83 total km: 960

https://mnfienen.github.io/talulat/2023/08/11/day10.html

**Comment by salajc26** [comment]
❤️❤️ keep on going Mike and Chandra.
I’m in Madison this weekend, hope to see you next time
